Workflows/Extensions/StringToInt.cs

```diff
@@ -0,0 +1,24 @@
+using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+
+[Combinator]
+[Description("")]
+[WorkflowElementCategory(ElementCategory.Transform)]
+public class StringToInt
+{
+   
+    public string[] Topic{ get; set;} 
```

Comment:
Since this is a collection of strings, the plural form is usually recommended, so `Topics` instead of `Topic`.